estate/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
+access_estate_property,access_estate_property,model_estate_property,base.group_user,1,1,1,0
```

Comment:
Does the tutorial ask to not give unlink (which is equivalent to delete) permissions ? (Referring to the last 0.)